contrib/docker/ckan-entrypoint.sh

```diff
@@ -58,29 +61,30 @@ link_redis_url () {
 
 # If we don't already have a config file, bootstrap
 if [ ! -e "$CONFIG" ]; then
+  write_config
+fi
```

Comment:
Then you just need to pass the variables $CKAN_SQLALCHEMY_URL, $CKAN_SOLR_URL or $CKAN_REDIS_URL to the container